degirum_tools/zone_count.py

```diff
@@ -215,7 +215,7 @@ def annotate(self, result, image: np.ndarray) -> np.ndarray:
         # draw annotations
         for zi in range(len(self._polygons)):
             cv2.polylines(
-                image, [self._polygons[zi]], True, line_color, result.overlay_line_width
+                image, self._polygons[zi].tolist(), True, line_color, result.overlay_line_width

```

Comment:
Well, it looks like not equivalent change: it was single-element list containing polygon, now it is just polygon. `cv2.polylines` expects list of polygons

Reply:
Oh, I assumed because of the iteration on line216 that it was just polygon but yes you are right its a list of polygons.